SWADroid/src/main/res/xml/tables.xml

```diff
@@ -385,6 +385,134 @@
         obligatory="true"
         type="foreign-key" />
     </table>
+    <table
+        name="games"
+        new-in-version="19"
+        to-string="%name%" >
+        <field
+            name="id"
+            obligatory="true"
+            type="long" />
+        <field
+            name="userSurname1"
+            obligatory="true"
+            type="text" />
+        <field
+            name="userSurname2"
+            obligatory="true"
+            type="text" />
+        <field
+            name="userFirstName"
+            obligatory="true"
+            type="text" />
+        <field
+            name="userPhoto"
+            obligatory="true"
+            type="text" />
```

Comment:
Why is it necessary to store user data in this table? If necessary, why is this data stored in all rows of the table?

Is the user not always the same student who participates in the game or is it different user data? If it is different user data, why is it necessary in the game and why is it stored?